prediction_market_agent_tooling/markets/seer/seer_contracts.py

```
@@ -0,0 +1,83 @@
+        )
+    )
+    address: ChecksumAddress = Web3.to_checksum_address(
+        "0x83183da839ce8228e31ae41222ead9edbb5cdcf1"
+    )
+
+    @staticmethod
+    def build_market_params(
+        market_question: str,
+        outcomes: list[str],
+        opening_time: DatetimeUTC,
+        min_bond_xdai: xDai = xdai_type(10),
```

Ruff (0.8.2)

Do not perform function call `xdai_type` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)